scripts/safe_template_linter.py

```diff
@@ -1873,18 +1890,15 @@ def _get_contexts(self, mako_template):
                     # get media type (e.g. get text/javascript from
                     # type="text/javascript")
                     match_type = match_type.group()[6:-1].lower()
-                    if match_type not in javascript_types:
-                        # TODO: What are other types found, and are these really
```

Comment:
@nedbat: While I was adjusting this code anyway, I decided to actually deal with unknown types, rather than make an assumption.